edx/frontend-app-admin-portal
# feat: adds CSV export button to SubscriptionDetailPage

### Description

* refactors invite learners button to be in line with subscription title
* adds DowloadCsvButton in line with LicenseAllocationHeader SearchBar
* adds functionality to export CSV file with license data

### Ticket link

[ENT-3986](https://openedx.atlassian.net/browse/ENT-3986)

### Testing instructions

* Checkout branch, run `npm install`, run `npm start`
* Navigate to the admin portal subscription management detail page
* Click the 'Download CSV' button and observe the contents of the downloaded file

### Screenshots

![image](https://user-images.githubusercontent.com/58956521/112849224-d7714f00-9076-11eb-821f-5d444f08ad6f.png)

![image](https://user-images.githubusercontent.com/58956521/112869902-a8190d00-908b-11eb-97b2-fc65ddc85944.png)
